dlrover/python/elastic_agent/torch/training.py

```diff
@@ -946,14 +952,14 @@ def _stop_workers(
                     super()._stop_workers(worker_group, is_restart)
 
             signal.alarm(0)
-        except TimeoutError as te:
+        except RdzvTimeoutError as te:
```

Comment:
Should not use 'RdzvTimeoutError' instead of 'TimeoutError' here. The implement here is not correlated with rendezvous.